File: src/backend/MyAIRunningMate.Python/app/domain/models/ingestion/user_metrics.py
from datetime import datetime
from typing import Optional
from pydantic import BaseModel

class UserMetrics(BaseModel):
    user_volumetric_oxygen_max: Optional[float] = None
    user_max_heart_rate: Optional[int] = None
    user_lactate_threshold_heart_rate: Optional[float] = None
    user_lactate_threshold_power: Optional[float] = None
    user_lactate_threshold_speed: Optional[float] = None
    user_beginning_body_battery: Optional[int] = None
    user_beginning_body_potential: Optional[int] = None

File: src/backend/MyAIRunningMate/MyAIRunningMate.Domain/Models/UserMetrics.cs
namespace MyAIRunningMate.Domain.Models;

public class UserMetrics
{
    public int Age { get; init; }
    public double WeightKg { get; init; }
    public double UserVolumetricOxygenMax { get; init; }
    public int UserMaxHeartRate { get; init; }
    public int UserLactateThresholdHeartRate { get; init; }
    public int UserLactateThresholdPower { get; init; }
    public double UserLactateThresholdSpeed { get; init; }
    public string UserVolumetricOxygenMaxRating { get; init; }
    public int FitnessPercentile { get; init; }
    public double PowerToWeightRatio { get; init; }
    public string PowerRating { get; init; }
    public double ThresholdPercentagePower { get; init; }
    public string TrainingLevel { get; init; }
    public string FitnessRankColor { get; init; }
}